src/Containers/StoreHoursList/StoreHoursList.tsx

```diff
@@ -0,0 +1,529 @@
+        if (parsedFromDate >= parsedToDate) {
+            setError('From time too big');
+            return null;
+        } 
+        timeObj = {
+            'to': to,
+            'from': from
+        };
+        return timeObj;
+    };
+    
+    const saveHours = (categoryName: string): void => {
+        if (Object.values(checkboxes).includes(true)) {
+            const timedArray = allCategoriesWithHours.find((categorySchedule: ICategoryWithHoursTypes): ICategoryWithHoursTypes | null | boolean => categorySchedule.category === categoryName);
+            if (timedArray !== undefined) {
+                Object.entries(checkboxes).map((checkbox): ICategoryWithHoursTypes | null => {
+                    if (checkbox[CHECKBOX_TIME] && timedArray.hoursByDay[checkbox[CHECKBOX_DAY]].length >= 1) { // cannot have more than one time per day
+                        setError('Only one time allowed');
+                    } else if (checkbox[CHECKBOX_TIME] && timedArray.hoursByDay[checkbox[CHECKBOX_DAY]].length === 0) { // add time
+                        const convertedTime = convertDateToHours(storeHours);
+                        if (convertedTime !== null) {
+                            timedArray.hoursByDay[checkbox[CHECKBOX_DAY]].push({
+                                from: convertedTime.from,
+                                to: convertedTime.to
+                            });
+                            setCheckboxes(initialCheckboxState);
+                            return timedArray;
+                        }
+                    }
+                    return null;
+                })
+            }
+        }
+    };
+
+    const getActiveSchedule = (categoryName: string): ICategoryWithHoursTypes => {
+        setActiveCategory(categoryName);
+        const activeSchedule = allCategoriesWithHours.find((el): ICategoryWithHoursTypes | null | boolean => categoryName === el.category);
+        if (activeSchedule) {
+            return activeSchedule;
+        }
+        return activeCategorySchedule;
+    };
+
+    useEffect((): void => {
+        setActiveCategorySchedule(getActiveSchedule(activeCategory))
+    }, [activeCategorySchedule]);
+
+    return (    
+        <StoreHoursContainer>
+            <SettingsCard heading={constants.TITLES.HEADING} icon={BusinessTime}>
+                <ButtonsContainer>
+                    <StyledButton
+                        icon={Edit}
+                        onClick={(): void => {
+                            setError('');
+                            setEditModalState(!editModalState);
+                        }}
+                    >
+                        { constants.BUTTONS.EDIT }
+                    </StyledButton>
+                    <StyledButton onClick={(): void => setIs24(!is24)}> 
+                        { constants.BUTTONS.TOGGLE } 
+                    </StyledButton>
+                </ButtonsContainer>
+                <StyledHeading type='h6'>
```

Comment:
Is this h6 hierachically or are you doing it for just the size?

Reply:
This would be just for the size.

Reply:
use CSS, html is for structure and CSS is for styling.

Reply:
This is part of the Heading UI component as a prop. Would you still recommend to separate the two pieces of logic.

For reference this is the written Heading component:
```
enum elements {
    h1 = 'h1',
    h2 = 'h2',
    h3 = 'h3',
    h4 = 'h4',
    h5 = 'h5',
    h6 = 'h6',
}

export const Heading: React.FC<HeadingProps> = ({
    children,
    type = 'h1',
    size = elements[type] || elements.h1,
    ...props
}): React.ReactElement => (
    <TextLayout type={type} size={size} {...props}>
        {children}
    </TextLayout>
);

export default Heading;
```

Reply:
Hmm in that case, change the size... I wish I can remake the ui library again, but im not an active member anymore, and ralph would prob strangle me